ui/hooks/useIncrementedGasFees.js

```diff
@@ -56,10 +56,9 @@ function getHighestIncrementedFee(originalFee, currentEstimate) {
  * ).CustomGasSettings} - Gas settings for cancellations/speed ups
  */
 export function useIncrementedGasFees(transactionGroup) {
-  const { primaryTransaction } = transactionGroup;
+  const { primaryTransaction = {} } = transactionGroup;
```

Comment:
did this fix any issue in particular? Or is this just a general cleanup item

Reply:
removed 😄